### PR TITLE
changing para 5

### PR DESCRIPTION
**Pull request by velvelshteynberg**
Changed para 5. 
I think its really cool

### CODE REVIEW

File: sample.txt [hunk 1]
```diff
@@ -14,4 +14,4 @@ Sailor ______!
 Sailor ______!
 
 With secret powers all so new to her
-She is the one named ______.
+She is the one named I am not really sure what her name is .
```
Comment (elvispeter91) [inline sample.txt:17]: Me neither !!  👍

**Review by elvispeter91 — APPROVED** [review]
Approved @velvelshteynberg

File: sample.txt (hunk 1 above)
Comment (RonnieHoffer) [reply sample.txt:17]: I think it was Carmen Sandiego

**Review by RonnieHoffer — APPROVED** [review]

**Comment by RonnieHoffer** [comment]
My first merge!

**Pull request merged**